Instastories.py

```diff
@@ -13,6 +13,8 @@
 except ImportError as e:
     PRINT_TABLE = False
 
+
+DEFAULT_CACHED_IDS_PATH = "cached_ids.json"
```

Comment:
Why default? We can't change it for now... 
Move this to settings.py and get the value with settings.get('cached_id_path')